meta-dts-distro/recipes-support/mei-amt-check/mei-amt-check.bb

```diff
@@ -0,0 +1,25 @@
+DESCRIPTION = "AMT status checker for Linux"
+SECTION = "tools"
+HOMEPAGE = "https://github.com/mjg59/mei-amt-check"
+
+LICENSE = "GPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=39f5702df600d6d1b1d6d138dc62347d"
+
+SRC_URI = "git://github.com/mjg59/mei-amt-check;protocol=https;branch=master \
```

Comment:
@kotylamichal please fix syntax here, here example https://github.com/Dasharo/meta-dts-ce/pull/2/files#diff-1c81a7aabd49b3985d32e343cf1916180229ee8e270414c5c278ad1095858099R5

Reply:
done like in OE wiki